tests/acceptance/run-tests.js

```diff
@@ -0,0 +1,35 @@
+const createTestCafe = require('testcafe');
+const { PORT } = require('./constants');
+const yargs = require('yargs');
+
+const argv = yargs
```

Comment:
This is a cool solution! I didn't think about the fact that we run two separate instances of this script with different browsers

Reply:
Thanks. I figured it was a nice way of running the tests